ee/kafka_client/client.py

```diff
@@ -8,8 +8,9 @@
 from kafka import KafkaProducer as KP
 
 from ee.clickhouse.client import async_execute, sync_execute
+from ee.kafka_client import helper
```

Comment:
Why was this needed to be moved into our repo rather than using the package

Reply:
because of base64 encoded keys. Going to move everything to the vendored bits, but that'll be a cleanup task.

Reply:
oh got it